src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_help.py

```diff
@@ -163,6 +173,10 @@
 helps['storage file list'] = """
     type: command
     short-summary: List files and directories in the specified share.
+    parameters:
+        - name: --files-only
```

Comment:
Consider something like '--omit-directories' or '--suppress-directories'. For '--file-only' I would need to read the help text to know what that means. The alternative are (at least to mean) a little more self-explanatory.